src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1571,10 +1592,19 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
-
-    // TODO: Add desired media content for upload. For more information, see:
+    // TODO: Use `media` to add desired media content for upload. For more information, see:
```

Comment:
Similarly here. At the very least, I suggest a common structure with upload and download:
eg
```
// TODO: To do foo,.... use:
//   blah blah
// For xxxx more information, see
// http://
```
as above. Or having a different order like you have here. But I think both should have the same order.